### DISCUSSION TITLE
Add the chance to disable mod sounds

### DISCUSSION THREAD

**Discussion by Dieal** [ppy/osu#18594] [Ideas] [answered]
Before, while I was playing, I decided to use the "Perfect" mod with a beatmap. Every time I get a 50 /  100, I have to start again.  The problem is that every time I fail an annoying sound plays and I get a heart attack. So I'm asking if it could be possible to add an option for disabling mod sounds. Thanks

**Comment by integerrr** [comment, marked as answer]
That's the fail sound you're hearing not the mod. Don't think straight up disabling it will be made available but they are looking into perhaps lowering the volume of that slightly: https://github.com/ppy/osu/pull/15382#issuecomment-955913357

**Reply by Dieal** [reply]
Oh, didn't know, thanks

**Comment by peppy** [comment]
You can also apply a skin changing that sound.
